code/edgebridge/build.gradle

```diff
@@ -68,14 +69,15 @@ android {
 android.libraryVariants.all { variant ->
     tasks.withType(Javadoc) {
         source = [android.sourceSets.main.java.sourceFiles, android.sourceSets.phone.java.sourceFiles]
-        ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
+        ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdk}/android.jar"
 
         doFirst{classpath = files(variant.javaCompile.classpath.files) + files(ext.androidJar)}
 
         exclude "**/BuildConfig.java"
         exclude "**/R.java"
         options {
             links "https://developer.android.com/reference"
+            linksOffline "https://d.android.com/reference/", "https://d.android.com/reference/androidx/"
```

Comment:
Is this line just replacing `https://d.android.com/reference/` with `https://d.android.com/reference/androidx/` when linking offline?  I think this won't work if offline as the URI you are using needs network access.